state-chain/pallets/cf-ingress-egress/src/migrations/btc_deposit_channels.rs

```diff
@@ -90,6 +90,7 @@ impl<T: Config<Instance3, TargetChain = Bitcoin>> OnRuntimeUpgrade for Migration
 					expires_at: old_channel.expires_at,
 					action: old_channel.action,
 					boost_fee: 0,
+					boost_status: BoostStatus::NotBoosted,
```

Comment:
Do we need these migrations still? I added boost status to get the code to compile, but it would be more correct for this to have a separate (e.g. DepositChannelDetails_v3) structs since this migration only updates from v2 to v3. Easier to just delete these of course if we don't need them. Similarly, I see that `boost_fee` was added here too, but `boost_fee` was only introduced in v5.

Reply:
I discussed this with Dan yesterday and agreed that we can already remove these migrations (since we aready have a branch for 1.3), which makes things easy.

Reply:
@dandanlen are you going to remove stale migrations separately, or do you want this to be done in this PR (I don't think it would be a big deal to merge as is and remove them later IMO).

Reply:
Yep I'll deal with it.